src/apps/config/leader.lua

```diff
@@ -88,16 +91,37 @@ function Leader:set_initial_configuration (configuration)
    end
 end
 
-function Leader:start_worker()
-   return worker.start("follower", self.worker_start_code)
+function Leader:start_worker(cpu)
+   local start_code = { self.worker_start_code }
+   if cpu then
+      table.insert(start_code, 1, "print('Bound data plane to CPU:',"..cpu..")")
+      table.insert(start_code, 1, "require('lib.numa').bind_to_cpu("..cpu..")")
+   end
+   return worker.start("follower", table.concat(start_code, "\n"))
 end
 
 function Leader:stop_worker(worker)
+   if worker.cpu then self.cpuset:release(worker.cpu) end
    S.kill(worker.pid, 15)
 end
 
+function Leader:acquire_cpu_for_follower(id, app_graph)
+   local pci_addresses = {}
+   -- Grovel through app initargs for keys named "pciaddr".  Hacky!
```

Comment:
Maybe this should call out to support?

Have the generic support return nil so that it just selects any CPU and in the lwaftr have it get the pciaddress from the app_graph. I don't like assuming where in the arguments the pciaddress is and that it's named "pciaddr". WDYT?

Reply:
Good idea!  Filed #958.